### PR TITLE
[TaskRabbit.com] New ruleset

### PR DESCRIPTION
**Pull request by youdly**

### CODE REVIEW

**Comment by youdly** [comment]
Travis: `Unable to find image 'node:latest' locally`.

**Comment by youdly** [comment]
Rebase

File: src/chrome/content/rules/TaskRabbit.com.xml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+		- api2		(timeout)
+		- assets	(mismatch hostname, CN: *.s3.amazonaws.com)
+		- www.taskrabbit.comwww.taskrabbit.com	¹
+		- help		¹
+		- ios		¹
+		- m		¹
+		- www.refer	¹
+		- sendgrid-tracking	(mismatch hostname, CN: *.sendgrid.com)
+		- static	¹
+		- statys	(mismatch hostname, CN: *.statuspage.io)
```
Comment (J0WI) [inline src/chrome/content/rules/TaskRabbit.com.xml]: Typo (=> status)

File: src/chrome/content/rules/TaskRabbit.com.xml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+		- store		(mismatch hostname, CN: *.tr.co)
+		- styleguide	¹
+		- tech	(mismatch hostname, CN: www.github.com)
+		- tr-console	¹
+
+	¹ Mismatch hostname, CN: www.taskrabbit.com
```
Comment (J0WI) [inline src/chrome/content/rules/TaskRabbit.com.xml]: Wildcard DNS

**Comment by youdly** [comment]
Fixed typo and removed irrelevant (wildcard dns) comments.

**Comment by J0WI** [comment]
LGTM, but one of the cert just expired. Let's see if the fix it.

**Comment by youdly** [comment]
Removed it. The expiry is 2 week ago, which a bit too late.

**Pull request merged**